waku/v2/protocol/waku_rln_relay/waku_rln_relay_types.nim

```diff
@@ -77,49 +77,249 @@ type WakuRLNRelay* = ref object
   ethAccountPrivateKey*: Option[PrivateKey]
   rlnInstance*: RLN[Bn256]
   pubsubTopic*: string # the pubsub topic for which rln relay is mounted
-  # contentTopic should be of type waku_message.ContentTopic, however, due to recursive module dependency, the underlying type of ContentTopic is used instead
-  # TODO a long-term solution is to place types with recursive dependency inside one file 
-  contentTopic*: string 
+                       # contentTopic should be of type waku_message.ContentTopic, however, due to recursive module dependency, the underlying type of ContentTopic is used instead
+                       # TODO a long-term solution is to place types with recursive dependency inside one file
+  contentTopic*: string
   # the log of nullifiers and Shamir shares of the past messages grouped per epoch
   nullifierLog*: Table[Epoch, seq[ProofMetadata]]
 
 type MessageValidationResult* {.pure.} = enum
-    Valid, Invalid, Spam
+  Valid, Invalid, Spam
 
 # inputs of the membership contract constructor
 # TODO may be able to make these constants private and put them inside the waku_rln_relay_utils
-const 
-  MEMBERSHIP_FEE* = 5.u256
+const
+  MembershipFee* = 5.u256
```

Comment:
For interest's sake, was this `const` identifier changed to `PascalCase` by `nimpretty`? No strong feelings, but I indeed prefer PascalCase for constants as is done here, and this is also what the official style guide suggests: https://nim-lang.org/docs/nep1.html#introduction-naming-conventions

I know we've been very inconsistent with this, though, but would suggest that we try to remain consistent at least within a module (i.e. either PascalCase or CAPITAL_LETTERS in this one). We can then see it as a single refactoring task to refactor all `const`s across the codebase to `PascalCase`, once we agree on a convention.

Reply:
> For interest's sake, was this const identifier changed to PascalCase by nimpretty?

I changed it myself, it was referred to in some other parts of the code as `MembershipFee`, I  wanted them to be consistent so I changed it.

>  No strong feelings, but I indeed prefer PascalCase for constants as is done here, and this is also what the official style guide suggests: https://nim-lang.org/docs/nep1.html#introduction-naming-conventions

I also prefer the Camal case and almost followed this in my implementation. Will revert this one also back to Upper case.

Reply:
Addressed in https://github.com/status-im/nim-waku/pull/941/commits/3d1038a7789d18c868e18b3d328817952f035947